docs/samplesheets/validate_sample_sheet.md

```diff
@@ -26,4 +26,8 @@ See an example in the `input` field from the [example schema.json](https://raw.g
 }
 ```
 
+!!! warning
+
+    The `.fromSamplesheet` channel operator and `samplesheetToList` also validate the files before converting them. If you convert the samplesheet, you should not add a schema to the parameter corresponding to the samplesheet to keep your pipeline as efficient as possible.
+
```

Comment:
I wouldn't add this warning, it's better to validate the sample sheet before starting the pipeline execution, even if it is then validated twice.

Reply:
Is it okay if I change it to a note instead, I think it's useful information

Reply:
sounds good